### DISCUSSION TITLE
Version changes for Not()-function in IntVar

### DISCUSSION THREAD

**Discussion by disl** [google/or-tools#4320] [CP-SAT questions] [answered]
Sorry, can't find any documentation or discussion. 
How can I remove/rewrite red-marked errors in C# (see below)?

![image](https://github.com/user-attachments/assets/f19ac10e-33a4-4ee4-9285-91dae1bb68b2)

**Comment by lperron** [comment, marked as answer]
```
   BoolVar transVar ...
```

or 

```
  ILiteral transVar = ...
```
